aspnetcore/native-aot/index.md

```diff
@@ -0,0 +1,183 @@
+| MVC | | | <span aria-hidden="true">❌</span><span class="visually-hidden">Not supported</span> |
+| Blazor Server | | |<span aria-hidden="true">❌</span><span class="visually-hidden">Not supported</span> |
+| SignalR | | | <span aria-hidden="true">❌</span><span class="visually-hidden">Not supported</span> |
+| Authentication | | | <span aria-hidden="true">❌</span><span class="visually-hidden">Not supported</span> (JWT soon) |
+| CORS | <span aria-hidden="true">✔️</span><span class="visually-hidden">Fully supported</span> | | |
+| HealthChecks | <span aria-hidden="true">✔️</span><span class="visually-hidden">Fully supported</span> | | |
+| HttpLogging | <span aria-hidden="true">✔️</span><span class="visually-hidden">Fully supported</span> | | |
+| Localization | | |<span aria-hidden="true">❌</span><span class="visually-hidden">Not supported</span> |
+| OutputCaching | <span aria-hidden="true">✔️</span><span class="visually-hidden">Fully supported</span> | | |
+| RateLimiting | <span aria-hidden="true">✔️</span><span class="visually-hidden">Fully supported</span> | | |
+| RequestDecompression | <span aria-hidden="true">✔️</span><span class="visually-hidden">Fully supported</span> | | |
+| ResponseCaching | <span aria-hidden="true">✔️</span><span class="visually-hidden">Fully supported</span> | | |
+| ResponseCompression | <span aria-hidden="true">✔️</span><span class="visually-hidden">Fully supported</span> | | |
+| Rewrite | <span aria-hidden="true">✔️</span><span class="visually-hidden">Fully supported</span> | | |
+| Session | | |<span aria-hidden="true">❌</span><span class="visually-hidden">Not supported</span> |
+| Spa | | |<span aria-hidden="true">❌</span><span class="visually-hidden">Not supported</span> |
+| StaticFiles | <span aria-hidden="true">✔️</span><span class="visually-hidden">Fully supported</span> | | |
+| WebSockets | <span aria-hidden="true">✔️</span><span class="visually-hidden">Fully supported</span> | | |
+
+It is important to test your application thoroughly when moving to a native AOT deployment model to ensure that functionality observed during development (when the app is untrimmed and JIT-compiled) is preserved in the native executable. When building your application, keep an eye out for AOT warnings. An application that produces AOT warnings during publishing is not guaranteed to work correctly. If you don't get any AOT warnings at publish time, you should be confident that your application will work consistently after publishing for AOT as it did during your F5 / `dotnet run` develoment workflow.
+
+For more information on AOT warnings and how to address them see; [Introduction to AOT warnings](/dotnet/core/deploying/native-aot/fixing-warnings).
+
+## Known issues
+
+We are keeping track of a number of known issues with native AOT support in ASP.NET Core in [a GitHub issue](https://github.com/dotnet/core/issues/8288).
+:::moniker-end
```

Comment:
```suggestion

```
You don't have ` :::moniker range=">= aspnetcore-9.0"` as it's not needed so can't have an end.
File should end in blank line